docs/content.zh/docs/dev/table/common.md

```diff
@@ -40,68 +40,83 @@ Table API 和 SQL 程序的结构
 {{< tabs "62be8916-7ab4-4831-bd19-05c591181835" >}}
 {{< tab "Java" >}}
 ```java
+// Create a TableEnvironment for batch or streaming execution.
```

Comment:
add imports for every class you are using in examples (can be a star import of the API)